docs/modules/operation/pages/bsm/introduction.adoc

```diff
@@ -3,16 +3,15 @@
 = Business Service Monitoring
 
 While {page-component-title} detects issues in your network by device, interface or service, the Business Service Monitoring (BSM) takes it one step further.
-The BSM components let you monitor and model high-level Business Services (BS) and help you quickly identify the most critical problems affecting them.
-With the BSM feature it is possible to model a high level BS context around the technical Service Monitors provided in {page-component-title}.
-To indicate which BS is affected by events at the technical Service Monitors level, a BS Operational Status is calculated.
+With the BSM feature, it is possible to model high level business service contexts around the technical Service Monitors provided in {page-component-title} and quickly identify the most critical problems affecting them.
+To indicate which business service is affected by events at the technical Service Monitors level, a business service Operational Status is calculated.
 
 As an example, let's assume a company runs an online store.
 Customers log in, select items, place them in the shopping cart and checkout using a payment system.
 The whole service is provided by a few web servers and access data from databases.
 To monitor the status of the databases, a SQL service monitor on each database server is configured.
 For testing the web servers a HTTP service monitor is used for each of them.
-Covering the overall functionality a Page Sequence Monitor (PSM) is used to test the login, shop and payment workflow through the provided web portal.
+Covering the overall functionality, a Page Sequence Monitor (PSM) is used to test the login, shop and payment workflow through the provided web portal.
```

Comment:
```suggestion
Covering the overall functionality, a Page Sequence Monitor (PSM) tests the login, shop, and payment workflow through the provided web portal.
```
is it the login workflow, shop workflow and payment workflow, or the "login and shop and payment workflow"?
If the former we can just add a comma and pluralize workflow: "login, shop, and payment workflows"